applications/nrf_desktop/doc/usb_state.rst

```diff
@@ -37,6 +37,7 @@ The USB stack is selected by enabling one of the following Kconfig choice option
   The USB legacy stack is used by default.
 * :ref:`CONFIG_DESKTOP_USB_STACK_NEXT <config_desktop_app_options>`
   The option enables USB next stack (:kconfig:option:`CONFIG_USB_DEVICE_STACK_NEXT`).
+  This is the only USB stack that supports USB High-Speed and nRF54H20.
```

Comment:
```suggestion
  This is the only USB stack that supports USB High-Speed and the nRF54H20 DK.
```
not sure if this should say DK but feels like nRF54H20 is missing something

Reply:
SoC would be better here

Reply:
Or `platform` - to be consistent with the rest of documentation.